pkg/namespacestore/namespacestore.go

```diff
@@ -394,6 +413,23 @@ func RunCreateAzureBlob(cmd *cobra.Command, args []string) {
 	})
 }
 
+// RunCreateNSFS runs a CLI command
+func RunCreateNSFS(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	createCommon(cmd, args, nbv1.NSStoreTypeNSFS, func(namespaceStore *nbv1.NamespaceStore, secret *corev1.Secret) {
+		fsPath := util.GetFlagStringOrPrompt(cmd, "fs-path")
+		fsBackend, _ := cmd.Flags().GetString("fs-backend")
```

Comment:
Will need to add checks for type of `fsBackend`

Reply:
same, added enum validation instead of the cli.